yaml/buddy-error-codes.yaml

```diff
@@ -1042,6 +1044,36 @@ Errors:
     gui_layout: "warning_dialog"
     approved: false
 
+  - code: "XX538"
+    title: "..." # Will be filled out with crash dump information
+    text: "..." # Will be filled out with crash dump information
+    id: "BSOD"
+    gui_layout: "red_screen"
+    approved: false
+
+  - code: "XX539"
+    title: "STACK OVERFLOW"
+    text: "..." # Will be filled out with crash dump information
+    id: "STACK_OVERFLOW"
+    gui_layout: "red_screen"
+    approved: false
+
+  - code: "XX540"
+    title: "WATCHDOG RESET"
+    text: "..." # Will be filled out with crash dump information
+    id: "WATCHDOG_RESET"
+    gui_layout: "red_screen"
+    approved: false
+
+  - code: "XX541"
+    title: "UNKNOWN ERROR"
+    text: "The printer has encountered an unrecognised error. Please contact the support."
```

Comment:
Do we want to instruct people to contact the support when they don't even have a dump that we could investigate?

Reply:
This fallback message will pop up when loading from crash dump failed. I'm not sure we won't be able to recover anything in this case, so contacting support incentivize to share the step-by-step process leading to that error.